src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/ContentEncryptionAlgorithmTests.cs

```diff
@@ -229,7 +206,6 @@ private static void VerifyAlgorithm3Des(byte[] encodedMessage)
             Assert.Equal(192, algorithm.KeyLength);
         }
 
-        [ConditionalFact(nameof(SupportsRc4))]
         public static void DecodeAlgorithmRc4_RoundTrip()
```

Comment:
`[Fact]` missing

Reply:
Of course, bad find and replace. I will fix it as soon as I get back to the office.